examples/tv-casting-app/tv-casting-common/tv-casting-app.matter

```diff
@@ -1854,7 +1854,7 @@ client cluster TargetNavigator = 1285 {
   command NavigateTarget(NavigateTargetRequest): NavigateTargetResponse = 0;
 }
 
-server cluster TestCluster = 1295 {
+server cluster TestCluster = 4293985551 {
```

Comment:
Why is this even implementing TestCluster?  Please file a followup to get that fixed and ping @chrisdecenzo about it?

Reply:
https://github.com/project-chip/connectedhomeip/issues/19172 flagged this one as well. It has a lot of TestCluster errors